Automattic/wp-calypso
# Seller Experience: Add redirection to site editor for "Start simple"

#### Changes proposed in this Pull Request

* Add redirection to `/site-editor/[siteSlug]` when choosing the Simple route
* New sites will use the default theme assigned during signup, Zoologist
* If you're going through the flow with an existing site that uses a non-FSE compatible theme, you'll be brought to the page editor's homepage instead.

**Video demo:**

https://user-images.githubusercontent.com/2124984/152184572-4b943d85-e676-442c-b6e4-ce1a2d18594f.mov


#### Testing instructions

* Switch to this PR, go to `/start`, and go through the signup process
* When you get to the `/setup-site/intent` step, add the feature flag to the URL: `&flags=seller-experience` to show the option for an online shop
* Choose Start Selling
* Add site title and tagline
* Choose "Start simple"
* You should be brought to the site editor with the Zoologist theme active
* Try the flow with an existing site that has a non-FSE theme active (like Karuna) by adding the site URL to the setup URL; `/start/setup-site/intent/yoursite.wordpress.com?flags=seller-experience`
* You should be brought to the page editor rather than the site editor when clicking "Start simple"

Related to #60717
